arch/arm/boot/dts/overlays/README

```diff
@@ -2000,6 +2000,10 @@ Params: spi-present-mask        8-bit integer, bitmap indicating MCP23S17 SPI0
                                 0-3, which can be configured with JP1 and JP2.
 
 
+Name:	pifi-40
+Info:	Configures the PiFi 40W stereo amplifier
+Load:	dtoverlay=pifi-40
+
```

Comment:
The formatting rules for the README are very strict:
* No hard TABs - indent with spaces, to column 8 or 32 (or 9/33 if numbering from 1) 
* No trailing whitespace
* All entries must have Name, Info, Load and Params.
* 2 blank lines between entries